### PR TITLE
fix(case overview) Fixing BIZ Data table headers not being shown

### PR DESCRIPTION
**Pull request by DumitruCorini**
Covers [BPO-190](https://bonitasoft.atlassian.net/browse/BPO-190)

### CODE REVIEW

File: uid-pages/form-case-overview/src/caseoverview.json [hunk 1]
```diff
@@ -1072,19 +1073,14 @@
...
-    },
     "case" : {
       "type" : "expression",
       "value" : "return  (!$data.openCase && $data.archivedCase.length > 0) ? $data.archivedCase[0] : $data.openCase;",
       "exposed" : false
     },
     "businessDataKeys" : {
       "type" : "expression",
-      "value" : "var businessData = $data.selectedBusinessData.length ? $data.selectedBusinessData[0] :  $data.selectedBusinessData;\n\nreturn Object.keys(businessData).filter(function(header){\n    return  header  !== \"persistenceId\" \n            && header.indexOf(\"persistenceVersion\") === -1 \n            && header !== \"links\";\n});",
+      "value" : "function objectIdAsString(header, headers) {\n    return header.endsWith(\"_string\") && headers.includes(header.replace(\"_string\", \"\"));\n}\n\nvar businessData = $data.selectedBusinessData.length ? $data.selectedBusinessData[0] :  $data.selectedBusinessData;\n\nreturn Object.keys(businessData).filter(function(header, index, headers){\n    return  !objectIdAsString(header, headers);\n}).filter(function(header){\n    return  header !== \"persistenceId\"\n            && header.indexOf(\"persistenceVersion\") === -1 \n            && header !== \"links\" \n});",
```
Comment (julienmege) [inline uid-pages/form-case-overview/src/caseoverview.json]: "objectIdAsString" could be renamed as "isObjectIdAsString" with some comment

Comment (julienmege) [inline uid-pages/form-case-overview/src/caseoverview.json]: Adding a test to validate:
- all the required header are displayed 
- we do not filter header like myheader_string

Comment (DumitruCorini) [reply uid-pages/form-case-overview/src/caseoverview.json]: done

Comment (DumitruCorini) [reply uid-pages/form-case-overview/src/caseoverview.json]: done

**Comment by julienmege** [comment]
🥇 LGTM,
Need to be merged after the release 7.9.2

**Pull request merged**